test-requirements.txt

```diff
@@ -0,0 +1,2 @@
+# https://github.com/OCA/stock-logistics-warehouse/pull/1566
+odoo-addon-stock-move-common-dest @ git+https://github.com/acsone/stock-logistics-warehouse.git@16.0-mig-stock_move_common_dest-hda#subdirectory=setup/stock_move_common_dest
```

Comment:
@glitchov Please put this in a separate commit to easy removal.

Reply:
Could you please also add the depending PR on this PR description in order to easy reviewers job ? Thanks